queue/src/fiskaltrust.Middleware.Contracts/Interfaces/ISigningDevice.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace fiskaltrust.Middleware.Contracts.Interfaces
+{
+    public interface ISigningDevice
+    {
+        public Task<bool> IsSigningDeviceAvailable();
+    }
```

Comment:
maybe call this `ISSCD` to have the same naming as in the interface? and then `IsSSCDAvailable` (or only `IsAvailable` 🤷)